deployments/config/user-service.yml

```diff
@@ -1,7 +1,18 @@
 # https://doc.traefik.io/traefik/routing/services/#configuration-examples
 http:
+  routers:
+    user-service-https:
+      entryPoints:
+        - "websecure"
+      rule: 'Host(`api.{{ env "DOMAIN_NAME" }}`) && PathPrefix(`/v1/users`)'
```

Comment:
Added a router for the `user-service`. This is similar to what is done for the `galactic-sovereign-service`. We can properly enforce the rate limiting.

It's a bit unclear if this was already applied before.